picoscope/ps4000.py

```diff
@@ -142,6 +142,13 @@ def __init__(self, serialNumber=None, connect=True):
             self.lib = windll.LoadLibrary(str(self.LIBNAME + ".dll"))
 
         super(PS4000, self).__init__(serialNumber, connect)
+        # check to see which model we have and use special functions if needed
+        if self._lowLevelGetUnitInfo(3) == '4262':
+            self.getTimestepFromTimebase = self.getTimestepFromTimebase4262
+            self.getTimeBaseNum = self.getTimeBaseNum4262
+        else:  # some other 4000 series unit
```

Comment:
Should be `elif unit_number in {'4223', '4224', '4423', '4424'}:`

Reply:
Are we sure the old calculations don't apply to any other series 4000 scopes?

Reply:
Yeah I read up on the datasheet. There are these two other scopes that have yet an other calculation. Picotech should really just write these functions in their driver for you.